README.md

```diff
@@ -105,13 +130,17 @@ export TM_SMTP_PORT=smtp-server-port-here
 export TM_SMTP_USER=smtp-server-user-here
 export TM_SMTP_PASSWORD=smtp-server-password-here
 ```
+
 </details>
 
 <details>
    <summary><bold>Set up the environment on Windows</bold></summary>
 
 ```bash
-setx TM_DB "postgresql://USER:PASSWORD@HOST/DATABASE"
+setx TM_APP_BASE_URL="web-server-endpoint"
```

Comment:
the equals sign on the setx commands should be a blank space instead

Reply:
Thanks for spotting! Fixed.